src/components/Components/ComponentBuildTrigger.tsx

```diff
@@ -4,31 +4,37 @@ import { PACState } from '../../hooks/usePACState';
 
 type Props = {
   pacState: PACState;
+  withoutTooltip?: boolean;
 };
 
-const ComponentBuildTrigger: React.FC<React.PropsWithChildren<Props>> = ({ pacState }) => {
-  if (pacState === PACState.disabled) {
-    return (
-      <Tooltip
-        trigger="click"
-        content="Trigger a new build manually from the component’s action menu. To enable an automatic trigger with every commit, upgrade the build pipeline plan to custom."
-      >
-        <Button variant={ButtonVariant.link} isInline>
-          Manual
-        </Button>
-      </Tooltip>
-    );
-  }
+const wrapInButtonTooltip = (triggerText: string, pacState: PACState) => {
+  const tooltipText =
+    pacState === PACState.disabled
+      ? 'Trigger a new build manually from the component’s action menu. To enable an automatic trigger with every commit, upgrade to the Custom build pipeline plan.'
+      : 'A new build pipeline run is automatically triggered with every commit to the source code.';
   return (
-    <Tooltip
-      trigger="click"
-      content="A new build pipeline run is automatically triggered with every commit to the source code."
-    >
+    <Tooltip trigger="click" content={tooltipText}>
       <Button variant={ButtonVariant.link} isInline>
-        Automatic
+        {triggerText}
       </Button>
     </Tooltip>
   );
 };
 
+const getTriggerType = (pacState: PACState) => {
+  return pacState === PACState.disabled ? 'Manual' : 'Automatic';
+};
+
+const ComponentBuildTrigger: React.FC<React.PropsWithChildren<Props>> = ({
+  pacState,
+  withoutTooltip = false,
+}) => {
+  const buildTriggerText = getTriggerType(pacState);
+  return withoutTooltip ? (
+    <span>{buildTriggerText}</span>
+  ) : (
+    wrapInButtonTooltip(buildTriggerText, pacState)
```

Comment:
this is becoming a component in itself. Would it be better to use the function for returning the `text` only?